src/azure-cli/azure/cli/command_modules/eventgrid/_params.py

```diff
@@ -153,24 +153,30 @@
 phone_extension_type = CLIArgumentType(
     help='The extension of the customer service number of the publisher. Only digits are allowed and number of digits should not exceed 10.')
 
+storage_queue_msg_ttl = CLIArgumentType(
+    help="Storage queue message time to live in seconds.",
+    options_list=['--storage-queue-msg-ttl', '--qttl'],
+    is_preview=True
+)
```

Comment:
You could add `type=int` definition here. There're two benefits:
- CLI with help check the input is valid int or not
- You don't need to use `int(storage_queue_msg_ttl)` to transform value in `custom.py` anymore